golem-worker-executor-base/src/services/oplog/mod.rs

```diff
@@ -132,6 +135,17 @@ pub trait OplogService: Debug {
     ) -> Result<(ScanCursor, Vec<OwnedWorkerId>), GolemError>;
 }
 
+/// Level of commit guarantees
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum CommitLevel {
+    /// Always commit immediately and do not return until it is done
+    Immediate,
+    /// Always commit, both for durable and ephemeral workers, no guarantees that it awaits it
+    Always,
+    /// Only commit immediately if the worker is durable
+    DurableOnly,
```

Comment:
is this also async / no guartantees? just thinking about the naming

Reply:
For durable workers all commits are as before - they are "async blocking" until the entries are written.
For ephemeral workers, the three levels are:
- like the above
- commit, but on a spawned background fiber
- ignore

tried to bring in names that makes sense, maybe failed :)